src/wallet/rpcwallet.cpp

```diff
@@ -1616,12 +1672,16 @@ UniValue listsinceblock(const UniValue& params, bool fHelp)
 
     if (fHelp)
         throw runtime_error(
-            "listsinceblock ( \"blockhash\" target-confirmations includeWatchonly)\n"
+            "listsinceblock ( \"blockhash\" target-confirmations includeWatchonly asOfHeight )\n"
             "\nGet all transactions in blocks since block [blockhash], or all transactions if omitted\n"
             "\nArguments:\n"
             "1. \"blockhash\"   (string, optional) The block hash to list transactions since\n"
             "2. target-confirmations:    (numeric, optional) The confirmations required, must be 1 or more\n"
             "3. includeWatchonly:        (bool, optional, default=false) Include transactions to watchonly addresses (see 'importaddress')"
+            "4. asOfHeight       (numeric, optional) Execute the query as if it were run when\n"
+            "                    the blockchain was at the height specified by this argument.\n"
+            "                    The default is to use the entire blockchain that the node is\n"
+            "                    aware of.\n"
```

Comment:
Document what happens if `asOfHeight` is greater than the height of the last known block.